pkg/collector/corechecks/servicediscovery/events_test.go

```diff
@@ -61,6 +61,7 @@ func Test_telemetrySender(t *testing.T) {
 			Stat: procStat{
 				StartTime: uint64(now.Add(-20 * time.Minute).Unix()),
 				RSS:       500 * 1024 * 1024,
+				CPUUsage:  1.5,
```

Comment:
Can you also add add non-zero cpu usage values to impl_linux_test.go (the one outside the module directory), to test propagation of the value in the agent check? You can check the RSS PR for an example.